omero_figure/templates/figure/index.html

```diff
@@ -914,12 +914,12 @@ <h5 style="float: left">Add Labels</h5>
 
 
     <script src="{% static 'figure/3rdparty/json2.js' %}"></script>
-    <script src="{% static 'figure/3rdparty/underscore.js' %}"></script>
-    <script src="{% static 'figure/3rdparty/backbone.js' %}"></script>
-    <script src="{% static 'figure/3rdparty/raphael/raphael-min.js' %}"></script>
+    <script src="{% static 'figure/3rdparty/underscore-1.3.3.js' %}"></script>
+    <script src="{% static 'figure/3rdparty/backbone-1.0.0.js' %}"></script>
+    <script src="{% static 'figure/3rdparty/raphael-2.1.2/raphael-min.js' %}"></script>
     <script src="{% static 'figure/3rdparty/mousetrap-1.5.3/mousetrap.min.js' %}"></script>
     <script src="{% static 'figure/3rdparty/backbone.mousetrap/backbone.mousetrap.js' %}"></script>
-    <script src="{% static 'figure/3rdparty/bootstrap3/js/bootstrap.js' %}"></script>
+    <script src="{% static 'figure/3rdparty/bootstrap-3.0.0/js/bootstrap.js' %}"></script>
```

Comment:
You missed the same ```bootstrap-3.0.0 ``` rename on the bootstrap css (at top of the page) so the app looks kinda broken now.